lib/elixir/lib/list.ex

```diff
@@ -467,58 +467,66 @@ defmodule List do
       iex> List.ascii_printable?('abc' ++ [0], 2)
       true
 
-  Improper lists are not printable, even if made only of ascii characters:
+  Last element in an improper list renders them not printable, even if made only of ASCII characters:
```

Comment:
I am having trouble understanding the rewritten sentence. I will let others give their feedback too. :)

Reply:
Same here. I think the previous version was easier to understand. I would only change `ascii` -> `ASCII`

Reply:
Is there a specific term to refer to the last element in an improper list?
What I am trying to say is that if we reach this one element within `limit`, this the the one that renders it non-printable

Reply:
Note that I am correct the previous sentence, since it is not correct.
Improper lists are printable provided limit doesn't reach their last element

Reply:
It is the same as the other discussion. :) They are not printable. If you pass one, then it may work by accident, but it is not a guarantee our API provides.